guides/databases/phpmyadmin.md

```diff
@@ -0,0 +1,51 @@
+# Intro
+
+---
+
+>**PHPMyAdmin** is a free software tool written in PHP, intended to handle  the administration of MySQL over the Web. phpMyAdmin supports a wide range  of operations on MySQL and MariaDB. Frequently used operations (managing  databases, tables, columns, relations, indexes, users, permissions, etc) can  be performed via the user interface, while you still have the ability to  directly execute any SQL statement.
+
+### Features provided by the program include:
+
+
+- Web interface
+
+- MySQL and MariaDB database management
+
+- Import data from CSV and SQL
+
+- Export data to various formats: CSV, SQL, XML, PDF (via the TCPDF library), ISO/IEC 26300 OpenDocument Text and Spreadsheet, Word, Excel, LaTeX and others
+
+- Administering multiple servers
+
+- Creating PDF graphics of the database layout
+
+- Creating complex queries using query-by-example (QBE)
```

Comment:
What does QBE mean? Explain it.

Reply:
It's a query language used in relational databases that help users to perform queries by providing a simple UI. It is just an abstraction between the user and the real query that the database system will receive and will transform into a DML form such as SQL.